src/flow_dot_js.ml

```diff
@@ -338,6 +395,21 @@ let dump_types js_file js_content =
 
       js_of_json types_json
 
+let coverage js_file js_content =
+    let filename = File_key.SourceFile (Js.to_string js_file) in
+    let root = Path.dummy_path in
+    let content = Js.to_string js_content in
+    match parse_content filename content with
+    | Error _ -> failwith "parse error"
```

Comment:
this seems too easy to hit. I'd say make it return a JSON error instead

Reply:
maybe `raise`?